### PR TITLE
Added a Podcast resources

### LINKED ISSUE #2136: [Add] <Tech Podcast>
### Site URL

https://www.youtube.com/@AI.Overpowered

### Description of the above provided site

In this podcast, the hosts Varun Mayya & Tanmay Bhat talk about everything that we can think of about AI helping us understand the use cases of AI in different domains. 

### Category

Other

### Sub Category

podcast

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by HVbajoria**
Closes #2136 

## Changes proposed

Through this PR, I have added about two amazing Podcasts: Overpowered and WVFRM.

### CODE REVIEW

**Comment by Anmol-Baranwal** [comment]
@CBID2 
The PR number is #2135 & issue number is #2136

I think people are taking it granted, that we are allowing these again and again.

**Comment by CBID2** [comment]
> @CBID2 
> 
> The PR number is #2135 & issue number is #2136
> 
> 
> 
> I think people are taking it granted, that we are allowing these again and again.

In this case @Anmol-Baranwal, do what you must.

**Comment by HVbajoria** [comment]
Hello @Anmol-Baranwal @CBID2,
Please let me know if I have made a mistake. I am really sorry if I have caused an inconvenience I just tried to do what I saw with the past pull requests and then work accordingly to add some good content. 
Could you please guide me I would learn and not repeat it. 
Thank you

**Comment by HVbajoria** [comment]
Hello team,
I have got my mistake. I had to first create an issue and then ask you to assign it before creating a pull request for the same.
I am closing this pull request. Sorry for the inconvenience and thank you for pointing my mistake.

**Comment by Anmol-Baranwal** [comment]
@HVbajoria 
Don't delete the branch, we will reopen this when the issue is assigned. I don't mean to discourage you, but you see we are like getting so many PRs like this so it's frustrating. No need to apologize.

**Comment by HVbajoria** [comment]
> @HVbajoria Don't delete the branch, we will reopen this when the issue is assigned. I don't mean to discourage you, but you see we are like getting so many PRs like this so it's frustrating. No need to apologize.

Thank you for understanding. I will wait for the issue to be assigned.

File: database/other/podcasts.json [hunk 1]
```diff
@@ -19,6 +19,12 @@
...
-]
\ No newline at end of file
+  },
+  {
+    "name": "Overpowered",
+    "description": " In Overpowered, the hosts Varun Mayya & Tanmay Bhat talk about everything that you can imagine about AI helping you get insights into how AI can integrate in various domains.",
+    "url": "https://www.youtube.com/@AI.Overpowered",
+    "category": "other",
+    "subcategory": "podcasts"
+  },
```
Comment (Anmol-Baranwal) [inline database/other/podcasts.json:29]:
```suggestion
  }
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
It is properly mentioned in the contributing guidelines to check validity of json structure.

**Review by Anmol-Baranwal — DISMISSED** [review]
🚀 Looks good to me. 

> @CBID2 
Please review this.

File: database/other/podcasts.json [hunk 2]
```diff
@@ -19,6 +19,12 @@
     "url": "https://stackoverflow.blog/podcast/",
     "category": "other",
     "subcategory": "podcasts"
+  },
+  {
+    "name": "Overpowered",
+    "description": " In Overpowered, the hosts Varun Mayya & Tanmay Bhat talk about everything that you can imagine about AI helping you get insights into how AI can integrate in various domains.",
```
Comment (CBID2) [inline database/other/podcasts.json:25]:
```suggestion
    "description": " In Overpowered, the hosts Varun Mayya & Tanmay Bhat talk about everything that you can imagine about AI, helping you get insights into how this piece of tech can be integrated in various domains.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just needs some tweaking @HVbajoria

**Review by Anmol-Baranwal — DISMISSED** [review]
🚀 Looks good to me. 

> @CBID2 Please review.

**Comment by HVbajoria** [comment]
Thank you @Anmol-Baranwal

File: database/other/podcasts.json [hunk 3]
```diff
@@ -19,6 +19,12 @@
     "url": "https://stackoverflow.blog/podcast/",
     "category": "other",
     "subcategory": "podcasts"
+  },
+  {
+    "name": "Overpowered",
+    "description": " In Overpowered, the hosts Varun Mayya & Tanmay Bhat talk about everything that you can imagine about AI, helping you get insights into how this piece of tech can be integrated in various domains.",
```
Comment (aftabrehan) [inline database/other/podcasts.json:25]: Good resource. Just this starting space could to be removed.

```suggestion
    "description": "In Overpowered, the hosts Varun Mayya & Tanmay Bhat talk about everything that you can imagine about AI, helping you get insights into how this piece of tech can be integrated in various domains.",
```

**Review by aftabrehan — APPROVED** [review]
Looks good to me! 👍

**Comment by Anmol-Baranwal** [comment]
@CBID2 
The changes requested by you have been done. Please review.

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Pull request merged**